pkg/controllers/node/health/controller.go

```diff
@@ -140,6 +140,12 @@ func (c *Controller) deleteNodeClaim(ctx context.Context, nodeClaim *v1.NodeClai
 		metrics.NodePoolLabel:     node.Labels[v1.NodePoolLabelKey],
```

Comment:
Consider being less specific in this metric if we're now going to have a metric that gives us specific reasons attached to image ID